Cargo.toml

```diff
@@ -19,16 +19,18 @@ semver = "1.0"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 serde_with = { version = "3.12.0", features = ["base64"] }
+strum = { version = "0.27.0", features = ["derive"] }
 thiserror = "1"
 tokio = { version = "1", default-features = false, features = ["net"] }
 tracing = "0.1.40"
 url = "2"
 uuid = { version = "1.11.0", features = ["v4", "serde"] }
 
 [features]
-default = ["client"]
+default = ["client", "relay_protocol"]
```

Comment:
IMHO it's not worth bothering with features, especially if they are enabled by default. It just creates a huge testing matrix that we aren't going to do.

Reply:
Yes, I agree atm. I was considering implementing part of the protocol here, which would introduce dependencies we don't necessarily want downstream. I should have removed the feature after moving that stuff out of here.